src/writer.js

```diff
@@ -19,7 +19,13 @@ class Writer {
   }
 
   append (span) {
-    const trace = span.context()._trace
+    const spanContext = span.context()
+    const trace = spanContext._trace
+
+    if (spanContext._sampling !== undefined && spanContext._sampling.pleaseDrop === true) {
```

Comment:
`_sampling` should always be available on the context.

Reply:
The writer unit tests weren't setting it, I'll update the unit test then to add it?

Reply:
I see. So yes, I'd rather update the test than modify the code specifically for tests.